### DISCUSSION TITLE
Is the "Comments" header from Disqus?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#2457] [Q&A] [answered]
Something I wonder now when I implemented Disqus is, is that "Comments" header above the comment section from Disqus itself, or does Material for MkDocs provide it?

I talk about this part here:
![image](https://user-images.githubusercontent.com/11576465/111132424-9f97e100-8579-11eb-9c2d-8d33077c55ca.png)

I would like to customize it a bit by adding the [GitHub comment SVG](https://octicons-primer.vercel.app/octicons/comment-24) to it.

**Comment by squidfunk** [comment]
No, it's from the Material for MkDocs translations:

https://github.com/squidfunk/mkdocs-material/blob/f0e87564db5d7f4cb0cc426d56d297b67c854eaa/src/partials/integrations/disqus.html#L31

**Reply by Andre601** [reply]
Could I just add the `<svg>` tag here or would I need to adjust certain things?

**Reply by Andre601** [reply]
Looks like I can just add the svg, but that would show it as black and I'm not sure what exactly I could do to fix this..

**Reply by squidfunk** [reply]
You would need to use the masking technique (`mask-image`) as outlined in the [admonition example](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#custom-admonitions).

**Reply by Andre601** [reply]
I'm not quite sure how I would insert this into the h2 title section tho.

**Comment by Andre601** [comment]
I still try to apply what you suggested @squidfunk but I'm not sure if I do anything right...

If I understood you correctly would I f.e. apply this here?
```css
.md-typeset h2 > .disqus-comments::before {
    background-color: rgb(255, 255, 255);
    -webkit-mask-image: var(--md-text-icon--comment);
            mask-image: var(--md-text-icon--comment);
}
```

This *should* to my understanding override any h2 of md-typeset that has a div or similar with the disqus-comments class to display the icon before it?

The HTML (`disqus.html`) looks like this (Just showing the relevant part. Everything else is from the partials):
```html
<!-- Disqus integration -->
{% if not page.is_homepage and disqus %}
  <h2 id="__comments"><div class="disqus-comments">{{ lang.t("meta.comments") }}</div></h2>
```

Trying stuff like applying the class to the header directly has no effect.

Like saying that I should apply the masking is easy to say, but for a CSS beginner like me is that equal to saying that I should try and build a NASA rocket.
I would appreciate some help in understanding this entire situation here.

**Reply by squidfunk** [reply]
Thanks for sharing your progess! I'm afraid I cannot assist you in designing a solution, as I'm trying to invest my time in making progress on Material for MkDocs itself. Maybe somebody else found a way to do this. My understanding was that you could start with the custom admonition example and work from there. I understand though that CSS is tricky sometimes.

**Reply by squidfunk** [reply]
Another possible way to solve this – you can use theme extension and include the icon directly in the template. You can then add classes to change the color:

https://github.com/squidfunk/mkdocs-material/blob/aac1eed0e1ed8a5bb94bfe2205367f1db1209385/src/overrides/main.html#L74-L93

**Reply by Andre601** [reply]
Thanks for that. Just putting it in using `{% include ".icons/octicons/comment-discussion-24.svg" %}` and surrounding this with the twemoji class helped already and applied the right color.

**Comment by Andre601** [comment, marked as answer]
## Solution
(Because you can't accept comments to answers as solutions)

```html
  <h2 id="__comments">
    <span class="twemoji">
      {% include ".icons/octicons/comment-discussion-24.svg" %}
    </span>
    {{ lang.t("meta.comments") }}
  </h2>
```

**Comment by Andre601** [comment]
I quickly ask this here because I don't think this is worth a own discussion.

Can the footnotes section be overridden using theme extension? I would like to add a "References" header to it, so that it looks a bit more similar to Wikipedia's footnote setup.

**Reply by squidfunk** [reply]
I don't think so. It's generated by Markdown, not the theme. However, you could consult the [documentation](https://python-markdown.github.io/extensions/footnotes/) or ask on the maintainers of Python Markdown.

**Reply by Andre601** [reply]
I found something.
The Footnotes extension allows you to use `///Footnotes Go Here///` (Or a custom text if you change in the settings) to change the position of footnotes.
